transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/widget/DateWidget.java

```diff
@@ -116,7 +116,7 @@ public void onUpdate(@Nullable final String selectedDate) {
             try {
                 mEditText.setText(mDateUtils.convertDateFromServerToWidgetFormat(selectedDate));
                 mListener.saveTextChanged(getName(), getValue());
-                mListener.valueChanged();
+                mListener.valueChanged(this);
```

Comment:
Keep it consistent with the other classes. Stick with either <Class>.this or just this to all calls to valueChanged